NebulaHost/PlayerManager.cs

```diff
@@ -205,7 +205,7 @@ public Player PlayerConnected(NebulaConnection conn)
         {
             //Generate new data for the player
             ushort playerId = GetNextAvailablePlayerId();
-            Float3 randomColor = new Float3(Random.value, Random.value, Random.value);
+            Float3 randomColor = new Float3(0f, 0f, 0f);
```

Comment:
This is only necessary for development builds of Unity which for some reason errors if we try to access UnityEngine.Random from here. It will be replaced by https://github.com/hubastard/nebula/pull/286 anyway